server/issue.go

```diff
@@ -628,6 +628,12 @@ func getIssueCustomFieldValue(issue *jira.Issue, key string) StringSet {
 		// Checkboxes, multi-select dropdown
 		result := NewStringSet()
 		for _, v := range value {
+			s, ok := v.(string)
```

Comment:
@levb In the case that the Jira field is a `[]string`, the type we have from the parsed Jira webhook payload is `[]interface{}`. Is there a practical way of doing a type assertion for `[]interface{}` -> `[]string`? I am instead checking the type of the element during iteration.

For context, we are grabbing the given Jira field without knowing the type beforehand. I recommend looking at this entire function (it's small).

Reply:
Nope. You could wrap it into a `NewStringSetWithArgs(args ...interface{})` which would do the same (and also try .String() if available?)

Reply:
The issue with that is we don't know if it is a `[]string`. It could be a `[]map`, which we then want to grab the `id` from each map. I can see `NewStringSetWithArgs` being used, if it returned `nil` when the parameter is not a `[]string`.

Reply:
Not sure I fully understand, I'm ok if you leave the code as is here.